pkgs/development/libraries/template-glib/default.nix

```diff
@@ -13,8 +13,8 @@ stdenv.mkDerivation {
     sha256 = "nsm3HgTU9csU91XveQYxzQtFwGA+Ecg2/Hz9niaM0Ho=";
   };
 
-  buildInputs = [ meson ninja pkg-config gettext flex bison vala glib gtk-doc docbook_xsl docbook_xml_dtd_43 ];
-  nativeBuildInputs = [ glib gobject-introspection ];
+  buildInputs = [ ninja pkg-config gettext flex bison vala glib gtk-doc docbook_xsl docbook_xml_dtd_43 ];
+  nativeBuildInputs = [ glib gobject-introspection meson ];
```

Comment:
Here it looks like `nativeBuildInputs` and `buildInputs` are swapped. With the exception of `gobject-introspection` that belongs to `nativeBuildInputs`.